integration-test/628-standardize-water-kinds.py

```diff
@@ -23,3 +23,33 @@ def test_reservoir(self):
         self.assert_has_feature(
             16, 10481, 25324, 'water',
             {'kind': 'lake', 'reservoir': True})
+
+    # this test is for https://github.com/tilezen/vector-datasource/pull/2047
+    def test_water_reservoir(self):
```

Comment:
@nvkelso please advice if we need some non-fake data and appreciate it if you can let me know which ones.

Also please advice whether we should add these tests here in this file or other issue number.

Reply:
It's ok, but a separate new test would be better since you're not otherwise modifying any of the other tests in this old file.

Reply:
I mentioned in the general PR, but responding directly here too. 

Please also add a test for the low zoom removals.

Clear Lake in California: [-4046481](https://www.openstreetmap.org/relation/4046481) with an area of 265548000. You can copy the props but just use DSL to make a box that's as large as that area instead of copying all the detailed polygon in.

That features name was being stripped until zoom 12 in error, but with this change it should be visible in a zoom from zoom 5+ depending on it's area, which looks like results in a min_zoom of 6. Since it's OSM it'll not actually be in zoom 6, so test a zoom 10 tile and see if that feature is there with that min_zoom.

This is the 256px tile I'd expect to see such a label in:

https://tile.openstreetmap.org/10/162/391.png

Reply:
Please also add a test for the low zoom removals.

Clear Lake in California: [-4046481](https://www.openstreetmap.org/relation/4046481) with an area of 265548000. You can copy the props but just use DSL to make a box that's as large as that area instead of copying all the detailed polygon in.

That features name was being stripped until zoom 12 in error, but with this change it should be visible in a zoom from zoom 5+ depending on it's area, which looks like results in a min_zoom of 6. Since it's OSM it'll not actually be in zoom 6, so test a zoom 10 tile and see if that feature is there with that min_zoom.

This is the 256px tile I'd expect to see such a label in:

https://tile.openstreetmap.org/10/162/391.png

Reply:
(It's OK but not great to edit the old test – but since you're not fixing anything in the old test, it's better to make a new file with the PR number in the name.)

Reply:
fixed in the latest commit